rent-book-api/spec/factories/resgister_book_detail.rb

```diff
@@ -0,0 +1,10 @@
+require "faker"
+
+FactoryBot.define do
+  factory :register_book_detail do
+    register_book_id{register_book_id}
+    book_id{book_id}
```

Comment:
```suggestion
    register_book
    book
```
khi factory các quan hệ có thể dùng thế này cho gọn, rồi khi gọi trong file spec thì dùng như cách a cmt ở trên
tương tự cho các model khác

Reply:
Dạ oke, Tks anh